bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-messages/container.jsx

```diff
@@ -23,10 +28,33 @@ const UserMessagesContainer = () => {
   return <UserMessages {...{ activeChats, roving }} />;
 };
 
-export default withTracker(() => {
+const Container = withTracker(() => {
   // Here just to add reactivity to this component.
   // We need to rerender this component whenever this
   // Storage variable changes.
   Storage.getItem(CLOSED_CHAT_LIST_KEY);
   return {};
 })(UserMessagesContainer);
+
+
+const blank = () => (
```

Comment:
I don't think thats needed for that case, we made it on userlist to demonstrations, but now with its merge we return the ts component directly